### PR TITLE
remove some redundant variables

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: lib/PuppeteerSharp/CDPSession.cs [hunk 1]
```diff
@@ -198,7 +198,6 @@ internal void OnMessage(ConnectionResponse obj)
             else
             {
                 var method = obj.Method;
-                var param = obj.Params?.ToObject<ConnectionResponseParams>();
```
Comment (SimonCropp) [inline lib/PuppeteerSharp/CDPSession.cs:201]: not sure on this one?

File: lib/PuppeteerSharp.Tests/InputTests/PageWaitForFileChooserTests.cs [hunk 2]
```diff
@@ -44,26 +44,26 @@ public async Task ShouldWorkWhenFileInputIsNotAttachedToDOM()
         }
 
         [Fact]
-        public async Task ShouldRespectTimeout()
+        public Task ShouldRespectTimeout()
         {
-            var ex = await Assert.ThrowsAsync<TimeoutException>(() => Page.WaitForFileChooserAsync(new WaitForFileChooserOptions
+            return Assert.ThrowsAsync<TimeoutException>(() => Page.WaitForFileChooserAsync(new WaitForFileChooserOptions
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/InputTests/PageWaitForFileChooserTests.cs:49]: Let's make it an arrow function.

Comment (kblok) [reply lib/PuppeteerSharp.Tests/InputTests/PageWaitForFileChooserTests.cs:49]: Same across the class

File: lib/PuppeteerSharp.Tests/PageTests/Events/ErrorTests.cs [hunk 3]
```diff
@@ -19,10 +19,10 @@ public async Task ShouldThrowWhenPageCrashes()
         {
             string error = null;
             Page.Error += (sender, e) => error = e.Error;
-            var gotoTask = Page.GoToAsync("chrome://crash");
+            await Page.GoToAsync("chrome://crash");
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/PageTests/Events/ErrorTests.cs:22]: The idea is not to await the task, so we wait for the error while the task is being executed.

File: lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerConnectTests.cs [hunk 4]
```diff
@@ -111,7 +111,7 @@ public async Task ShouldBeAbleToReconnectToADisconnectedBrowser()
         public async Task ShouldBeAbleToConnectToTheSamePageSimultaneously()
         {
             var browserOne = await Puppeteer.LaunchAsync(new LaunchOptions());
-            var browserTwo = await Puppeteer.ConnectAsync(new ConnectOptions
+            await Puppeteer.ConnectAsync(new ConnectOptions
```
Comment (kblok) [inline lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerConnectTests.cs:114]: This test is wrong. According to this https://github.com/puppeteer/puppeteer/blob/054d782c823c363c418bf478b0dbefa26673b358/test/launcher.spec.ts#L592 we should create a page with browserTwo in line 125.

File: lib/PuppeteerSharp/CDPSession.cs (hunk 1 above)
Comment (kblok) [reply lib/PuppeteerSharp/CDPSession.cs:201]: interesting, but yeah, looks dead.

**Comment by kblok** [comment]
We're having some fails in AppVeyor

**Pull request closed without merging**